mpi-proxy-split/unit-test/Makefile

```diff
@@ -53,7 +53,8 @@ default: ${TEST_BINS}
 ../record-replay.o ../drain_send_recv_packets.o:
 	@make -C ..
 
-%.exe: %.o ../record-replay.o ../split_process.o ../lower-half/procmapsutils.o
+# UNCOMMENT the objects below if compiling with SINGLE_CART_REORDER
+%.exe: %.o ../record-replay.o # ../split_process.o ../lower-half/procmapsutils.o
```

Comment:
MANA won't be able to build when compiling with SINGLE_CART_REORDER. 
You need to keep them by default and add a comment that these object files are needed by this macro and they should be removed if we no longer use the single_cart_reorder implementation.